.github/workflows/main.yml

```diff
@@ -112,7 +112,9 @@ jobs:
           npm install -g yarn
       - name: Build and run zemu tests
         run: |
-          make test_all
+          for i in {1..3}; do
```

Comment:
I am curious about this. What is this for?

Reply:
this is to avoid some sporadic failures that might appear in Zemu tests